helpers/console_helper.rb

```diff
@@ -0,0 +1,25 @@
+module ConsoleHelper
+  EXIT_COMMAND = 'exit'.freeze
+  CONFIRM_YES = 'y'.freeze
+
+  def exit?(command)
+    command == EXIT_COMMAND
+  end
+
+  def confirm?(*args, **kwargs)
+    output(*args, **kwargs)
+    input == CONFIRM_YES
+  end
+
+  def leave_loop
+    raise StopIteration
+  end
+
+  def input
+    gets.chomp
```

Comment:
How about this?
```suggestion
  def input(*args, **kwargs)
    output(*args, **kwargs)
    gets.chomp
```
So you'll be able to
```ruby
  def password_input
    input('ask.password')
  end
```
instead of
```ruby
  def password_input
    output('ask.password')
    input
  end
```